### PR TITLE
Setup project and linters, create data model

### PR DESCRIPTION
**Pull request by Berkay-akbas**
# Hello Code Reviewer 👋 

# Changes ♻️ 
- Project initialized with Postgres database set up.
- Installed Rubocop and Stylelint.
- Schema builded with running migrations. Table and column names match the provided ERD diagram. Foreign keys are all included. All columns that are foreign keys have a corresponding index.

Thanks for reviewing my code!

### CODE REVIEW

File: db/migrate/20221018112002_create_comment.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+      t.integer :post_id
+      t.string :text
+      t.timestamps
+    end
+  end
```
Comment (alexander16108) [inline db/migrate/20221018112002_create_comment.rb:9]: for your tables, there should be a foreign key added to reference other tables  👍🏾

File: db/migrate/20221018112009_create_like.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
+      t.integer :post_id
+      t.timestamps
+    end
+  end
+end
```
Comment (alexander16108) [inline db/migrate/20221018112009_create_like.rb:9]: for your tables, there should be a foreign key added to reference other tables  👍🏾

File: db/migrate/20221018112015_create_post.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.string :title
+      t.text :text
+      t.integer :comments_count
+      t.integer :likes_count
+      t.timestamps
+    end
+  end
```
Comment (alexander16108) [inline db/migrate/20221018112015_create_post.rb:11]: for your tables, there should be a foreign key added to reference other tables  👍🏾

File: db/migrate/20221018112002_create_comment.rb (hunk 1 above)
Comment (Berkay-akbas) [reply db/migrate/20221018112002_create_comment.rb:9]: @alexander16108 Hello, those are added with another migration.

Comment (Berkay-akbas) [reply db/migrate/20221018112002_create_comment.rb:9]: You can see those in the migration folder.

**Comment by Berkay-akbas** [comment]
@alexander16108 Also if you check schema.rb in /db you could see that relations are in fact there.

**Review by ShahierNashaat — APPROVED** [review]
Hi @Berkay-akbas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️ 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**